dbms/mysql/mysqldump/src/cloud/benchflow/collectors/dump.go

```diff
@@ -67,6 +66,30 @@ func backupHandler(w http.ResponseWriter, r *http.Request) {
     // cmdd.Run()
     // cmdd.Wait()
     
+    for _,each := range tables {
+		cmd := exec.Command("mysqldump", "-h", os.Getenv("MYSQL_HOST"), "-P", os.Getenv("MYSQL_PORT"), "-u", os.Getenv("MYSQL_USER"), "-p" + os.Getenv("MYSQL_USER_PASSWORD"), os.Getenv("MYSQL_DB_NAME"), each)
+		outfile, err := os.Create("/app/backup.sql")
```

Comment:
Use a different file for each iteration, which name depends on the value of `each`, and delete it at the end of the iteration.